### PR TITLE
Move version into controller_config

### PR DESCRIPTION
**Pull request by zhiqiangjin**
Also change version information to "major + '.' + minor" format in related functions.

### CODE REVIEW

File: goaviatrix/version.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ func (c *Client) Upgrade(version *Version) error {
 	attachSpokeToTransitGw.Add("action", "upgrade")
 
 	if version.Version == "" {
-		return errors.New("no target version is set")
```
Comment (arvindsree) [inline goaviatrix/version.go]: I think this logic should be moved to terraform part and goaviatrix should return error

**Comment by arvindsree** [comment]
update erb file as well

**Comment by arvindsree** [comment]
and remove the version resource

File: aviatrix/data_source_aviatrix_caller_identity_test.go [hunk 2]
```diff
@@ -60,7 +60,7 @@ func testAccDataSourceAviatrixCallerIdentity(name string) resource.TestCheckFunc
 			return fmt.Errorf("valid CID was not returned. Get version API gave the following Error: %v", err)
 		}
 
-		if !strings.Contains(version, "UserConnect") {
+		if !strings.Contains(version, ".") {
```
Comment (arvindsree) [inline aviatrix/data_source_aviatrix_caller_identity_test.go]: why is this changed?

File: goaviatrix/version.go (hunk 1 above)
Comment (zhiqiangjin) [reply goaviatrix/version.go]: Agree.

File: aviatrix/data_source_aviatrix_caller_identity_test.go (hunk 2 above)
Comment (zhiqiangjin) [reply aviatrix/data_source_aviatrix_caller_identity_test.go]: "GetCurrentVersion" now only returns "major" + "." + "minor" without "UserConnect".
